packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/GroupedSeriesConfiguration.tsx

```diff
@@ -353,7 +359,8 @@ const GroupedSeriesConfiguration: FC<GroupedSeriesConfigurationProps> = ({
                                                         ref={innerRef}
                                                         {...draggableProps}
                                                     >
-                                                        <GroupedSeriesConfigWrapper
+                                                        <Box
+                                                            mb={5}
```

Comment:
same here, can we avoid 5 and stick with theme

Reply:
👍